src/Compilers/CSharp/Portable/Symbols/Source/SourceNamespaceSymbol.AliasesAndUsings.cs

```diff
@@ -721,14 +721,39 @@ private UsingsAndDiagnostics GetUsingsAndDiagnostics(ref UsingsAndDiagnostics? u
                                 continue;
                             }
 
+                            var flags = BinderFlags.SuppressConstraintChecks;
                             if (usingDirective.UnsafeKeyword != default)
-                                diagnostics.Add(ErrorCode.ERR_BadUnsafeInUsingDirective, usingDirective.UnsafeKeyword.GetLocation());
+                            {
+                                if (usingDirective.StaticKeyword == default)
+                                {
+                                    diagnostics.Add(ErrorCode.ERR_BadUnsafeInUsingDirective, usingDirective.UnsafeKeyword.GetLocation());
+                                }
+                                else
+                                {
+                                    // Make sure 'unsafe' is even allowed in this language version, reporting an error
+                                    // otherwise.  If it is available, ensure that the user passed the '/unsafe' flag to
+                                    // to the compiler.
+                                    if (MessageID.IDS_FeatureUsingTypeAlias.CheckFeatureAvailability(diagnostics, usingDirective, usingDirective.UnsafeKeyword.GetLocation()))
```

Comment:
Same comment here. I would not make `CheckUnsafeModifier` conditional on langver check

Reply:
done.

Reply:
Was a test affected?

Reply:
yes.  many tests were affected by this.